slices/libxml2.yaml

```
@@ -0,0 +1,11 @@
+package: libxml2
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libicu70_libs
+      - liblzma5_libs
+      - zlib1g_libs
+    contents:
+      /usr/lib/*-linux-*/libxml2.so.*:
```

```suggestion
      /usr/lib/*-linux-*/libxml2.so.2*:
```

Just to be a bit more restrictive and thus safer from possible future conflicts :) 

Apart from this, it LGTM

---

Thank you!